src/styles/components/_component-card.scss

```diff
@@ -43,10 +46,16 @@
       z-index: 1;
       font-size: 1.125rem; // 18px/16
       line-height: 4.6667rem; // 75px
-      letter-spacing: 0.02em;
-      border-top: 0.25rem solid map.get($colors, "black");
-      background-color: map.get($colors, "white");
+      background-color: var(--background-color);
       position: relative;
+      text-align: center;
+      display: block;
```

Comment:
We'll have to add the `letter-spacing: 0.02em` back in.

Reply:
It was 2%--should I add that back in or just update to the .02em?

Reply:
Oh sorry, I put this in a reply above!!

_So the 2% did come from Figma correct. However, Yosevu reminded me that percents done work for letter-spacing, so I grabbed the letter-spacing code from Figma, which was the 0.02em (not 2em)_